### PR TITLE
Add amazon DynamoDB connector example

### PR DESCRIPTION
**Pull request by KasunAratthanage**

### CODE REVIEW

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 1]
```diff
@@ -0,0 +1,945 @@
+# Amazon DynamoDB Connector Example
+
+Amazon DynamoDB Connector allows you to access the [Amazon DynamoDB REST API](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/HowItWorks.API.html) through WSO2 EI. 
+
+Amazon DynamoDB is a fully managed NoSQL database service that provides fast and predictable performance with seamless scalability. AmazonDynamoDB enables customers to offload the administrative burdens of operating and scaling distributed databases to AWS, so they don’t have to worry about hardware provisioning, setup and configuration, replication, software patching, or cluster scaling.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:5]:
```suggestion
Amazon DynamoDB is a fully managed NoSQL database service that provides fast and predictable performance with seamless scalability. AmazonDynamoDB enables customers to offload the administrative burdens of operating and scaling distributed databases to AWS, so they do not have to worry about hardware provisioning, setup and configuration, replication, software patching, or cluster scaling.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 2]
```diff
@@ -0,0 +1,945 @@
...
+2. Insert employee details (items) in to the created table.
+3. Update employee details table.
+4. Retrieve information about the inserted employee details (items).
+5. Remove inserted employee details (items).
+6. Retrieve list of tables.
+7. Remove created employee details table.
+
+All seven operations are exposed via an API. The API with the context `/resources` has seven resources
+
+* `/addtable` : Creates a new table in the Amazon DynamoDB with the specified table name for store employee details.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:23]:
```suggestion
* `/addtable` : Creates a new table in the Amazon DynamoDB with the specified table name to store employee details.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 3]
```diff
@@ -0,0 +1,945 @@
...
+* `/insertdetails` : Insert employee data (items) and stored into the specified table.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:24]:
```suggestion
* `/insertdetails` : Insert employee data (items) and store in the specified table.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 4]
```diff
@@ -0,0 +1,945 @@
...
+
+If you do not want to configure this yourself, you can simply [get the project](#get-the-project) and run it.
+
+## Configure the connector in WSO2 Integration Studio
+
+Connectors can be added to integration flows in [WSO2 Integration Studio](https://wso2.com/integration/integration-studio/), which is the tooling component of WSO2 EI. Once added, the operations of the connector can be dragged onto your canvas and added to your resources.
+
+### Import the connector
+
+Follow these steps to set up the ESB Solution Project and the Connector Exporter Project.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:47]:
```suggestion
Follow these steps to set up the Integration Project and the Connector Exporter Project. 
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 5]
```diff
@@ -0,0 +1,945 @@
...
+
+1. Initialize the connector.
+    
+    1. Navigate into the **Palette** pane and select the graphical operations icons listed under **Amazondynamodb Connector** section. Then drag and drop the `init` operation into the Design pane.
+        
+        <img src="../../../../assets/img/connectors/amazon-dynamodb-api-init.png" title="Drag and drop init operation" width="500" alt="Drag and drop init operation"/>   
+    
+    2. Add the property values into the `init` operation as shown below. Replace the `region`, `accessKeyId`, `secretAccessKey`, `blocking` with your values.
+        
+       - **region** : The region of the application access. .
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:71]:
```suggestion
       - **region** : The region of the application access.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 6]
```diff
@@ -0,0 +1,945 @@
...
+1. Initialize the connector.
+    
+    1. Navigate into the **Palette** pane and select the graphical operations icons listed under **Amazondynamodb Connector** section. Then drag and drop the `init` operation into the Design pane.
+        
+        <img src="../../../../assets/img/connectors/amazon-dynamodb-api-init.png" title="Drag and drop init operation" width="500" alt="Drag and drop init operation"/>   
+    
+    2. Add the property values into the `init` operation as shown below. Replace the `region`, `accessKeyId`, `secretAccessKey`, `blocking` with your values.
+        
+       - **region** : The region of the application access. . 
+       - **accessKeyId** : The AWS secret access key..
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:72]:
```suggestion
       - **accessKeyId** : The AWS secret access key.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 7]
```diff
@@ -0,0 +1,945 @@
...
+    
+        <img src="../../../../assets/img/connectors/amazon-dynamodb-api-init-operation.png" title="Add values to the init operation" width="800" alt="Add values to the init operation"/>
+
+2. Set up the createTable operation.
+
+    1. Navigate into the **Palette** pane and select the graphical operations icons listed under **Amazondynamodb Connector** section. Then drag and drop the `createTable` operation into the Design pane.
+           
+        <img src="../../../../assets/img/connectors/amazon-dynamodb-drag-and-drop-create-table.png" title="Drag and drop create table operation" width="500" alt="Drag and drop create table operation"/>    
+
+    2. The createTable operation creates a new table. Table names must be unique within each region. `createTable` operation parameters listed here.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:84]:
```suggestion
    2. The createTable operation creates a new table. Table names must be unique within each region. The `createTable` operation parameters are listed here.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 8]
```diff
@@ -0,0 +1,945 @@
...
+               
+       - **attributeDefinitions** : A list of attributes that describe the key schema for the table and indexes. If you are adding a new global secondary index to the table, AttributeDefinitions should include the key element(s) of the new index. 
+       - **tableName** : The name of the table to create. 
+       - **keySchema** : Specifies the attributes that make up the primary key for a table or an index. The attributes in keySchema must also be defined in attributeDefinitions.
+       - **localSecondaryIndexes** : One or more local secondary indexes (the maximum is five) to be created on the table. Each index is scoped to a given partition key value. There is a 10 GB size limit per partition key value. Else, the size of a local secondary index is unconstrained.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:89]:
```suggestion
       - **localSecondaryIndexes** : One or more local secondary indexes (the maximum is five) to be created on the table. Each index is scoped to a given partition key value. There is a 10 GB size limit per partition key value. Alternately, the size of a local secondary index is unconstrained. 
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 9]
```diff
@@ -0,0 +1,945 @@
...
+       - **localSecondaryIndexes** : One or more local secondary indexes (the maximum is five) to be created on the table. Each index is scoped to a given partition key value. There is a 10 GB size limit per partition key value. Else, the size of a local secondary index is unconstrained. 
+       - **provisionedThroughput** : Represents the provisioned throughput setting for a specified table or index.
+        
+        While invoking the API, the above five parameter values come as a user inputs.
+        
+        <img src="../../../../assets/img/connectors/amazon-dynamodb-drag-and-drop-create-table-parameters.png" title="Drag and drop create table operation" width="500" alt="Drag and drop create table operation"/> 
+    
+    3. To get the input values in to the API we can use the [property mediator](../../../mediators/property-Mediator). Navigate into the **Palette** pane and select the graphical mediators icons listed under **Mediators** section. Then drag and drop the `Property` mediators into the Design pane as shown bellow.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:96]:
```suggestion
    3. To get the input values in to the API we can use the [property mediator](../../../mediators/property-Mediator). Navigate into the **Palette** pane and select the graphical mediators icons listed under the **Mediators** section. Then drag and drop the `Property` mediators into the Design pane as shown below.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 10]
```diff
@@ -0,0 +1,945 @@
...
+   
+   You can use the same configuration to initialize the connector. Please follow the steps given in 1.1 for setting up the `init` operation to the addtable operation.
+   
+2. Set up the updateTable operation.
+
+    1. Navigate into the **Palette** pane and select the graphical operations icons listed under **Amazondynamodb Connector** section. Then drag and drop the `updateTable` operation into the Design pane.
+           
+        <img src="../../../../assets/img/connectors/amazon-dynamodb-drag-and-drop-update-table.png" title="Drag and drop create put items operation" width="500" alt="Drag and drop put items operation"/>    
+
+    2. The updateTable operation use to update the created tables. `updateTable` operation parameters listed here.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:186]:
```suggestion
    2. The updateTable operation is used to update the created tables. The `updateTable` operation parameters are listed here.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 11]
```diff
@@ -0,0 +1,945 @@
...
+               
+       - **provisionedThroughput** : The new provisioned throughput setting for the specified table or index. 
+       - **tableName** :  The name of the table to contain the item. 
+        
+        While invoking the API, the above two parameter values come as a user inputs.
+        
+        <img src="../../../../assets/img/connectors/amazon-dynamodb-drag-and-drop-update-table-parameters.png" title="Drag and drop put items table operation" width="500" alt="Drag and drop put items table operation"/> 
+    
+    3. Then drag and drop the `Property` mediators into the Design pane as mentioned in `addtable` operation. The parameters available for configuring the Property mediator are as follows.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:195]:
```suggestion
    3. Then drag and drop the `Property` mediators into the Design pane as mentioned in the `addtable` operation. The parameters available for configuring the Property mediator are as follows.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 12]
```diff
@@ -0,0 +1,945 @@
...
+   
+   You can use the same configuration to initialize the connector. Please follow the steps given in 1.1 for setting up the `init` operation to the addtable operation.
+   
+2. Set up the getItem operation.
+
+    1. Navigate into the **Palette** pane and select the graphical operations icons listed under **Amazondynamodb Connector** section. Then drag and drop the `getItem` operation into the Design pane.
+           
+        <img src="../../../../assets/img/connectors/amazon-dynamodb-drag-and-drop-get-item.png" title="Drag and drop create get items operation" width="500" alt="Drag and drop get items operation"/>    
+
+    2. The getItem operation use to retrive inserted items to the tables. `getItem` operation parameters listed here.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:221]:
```suggestion
    2. The getItem operation is used to retrieve inserted items to the tables. The `getItem` operation parameters are listed here.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 13]
```diff
@@ -0,0 +1,945 @@
...
+   
+   You can use the same configuration to initialize the connector. Please follow the steps given in 1.1 for setting up the `init` operation to the addtable operation.
+   
+2. Set up the deleteItem operation.
+
+    1. Navigate into the **Palette** pane and select the graphical operations icons listed under **Amazondynamodb Connector** section. Then drag and drop the `deleteItem` operation into the Design pane.
+           
+        <img src="../../../../assets/img/connectors/amazon-dynamodb-drag-and-drop-delete-item.png" title="Drag and drop create get items operation" width="500" alt="Drag and drop get items operation"/>    
+
+    2. The deleteItem operation use to remove inserted items from the table. `deleteItem` operation parameters listed here.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:256]:
```suggestion
    2. The deleteItem operation is used to remove inserted items from the table. The `deleteItem` operation parameters are listed here.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 1 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:5]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 2 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:23]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 3 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:24]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 4 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:47]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 5 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:71]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 6 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:72]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 7 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:84]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 8 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:89]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 9 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:96]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 10 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:186]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 11 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:195]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 12 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:221]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 13 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:256]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 14]
```diff
@@ -0,0 +1,945 @@
...
+    3. Then drag and drop the `Property` mediators into the Design pane as mentioned in `addtable` operation. The parameters available for configuring the Property mediator are as follows:
+                
+       Add the property mediator to capture the `tableName` value.  Please follow the steps given in `deleteTable` operation. Please follow the steps given in `listdetails` operation.
+   
+       - **name** : tableName
+       - **expression** : json-eval($.tableName)
+    
+#### Get a response.   
+ 
+When you are invoking the created API, the request of the message is going through the each resource. Finally, it is passed to the [Respond mediator](https://ei.docs.wso2.com/en/next/micro-integrator/references/mediators/respond-Mediator/). The Respond Mediator stops the processing on the current message and sends the message back to the client as a response.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:346]: Let's change this to a relative link.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 15]
```diff
@@ -0,0 +1,945 @@
...
+       - **exclusiveStartTableName** : The first table name that the listTables operation evaluates. Use the value returned for LastEvaluatedTableName. 
+       - **limit** : The maximum number of table names to retrieve. If this parameter is not specified, the limit is 100.
+       
+        While invoking the API, the above two parameter values come as a user inputs.
+        
+        <img src="../../../../assets/img/connectors/amazon-dynamodb-drag-and-drop-delete-table-parameters.png" title="Drag and drop list table parameter operation" width="500" alt="Drag and drop list table parameter operation"/> 
+    
+    3. Then drag and drop the `Property` mediators into the Design pane as mentioned in `addtable` operation. The parameters available for configuring the Property mediator are as follows:
+                
+       Add the property mediator to capture the `tableName` value.  Please follow the steps given in `deleteTable` operation. Please follow the steps given in `listdetails` operation.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:339]:
```suggestion
       Add the property mediator to capture the `tableName` value.  Please follow the steps given in the `deleteTable` and `listdetails` operations.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 16]
```diff
@@ -0,0 +1,945 @@
...
+    1. Navigate into the **Palette** pane and select the graphical operations icons listed under **Amazondynamodb Connector** section. Then drag and drop the `deleteTable` operation into the Design pane.
+           
+       <img src="../../../../assets/img/connectors/amazon-dynamodb-drag-and-drop-delete-table.png" title="Drag and drop list table operation" width="500" alt="Drag and drop list table operation"/>    
+
+    2. The listTables operation use to retrieve information about the created tables. `deleteTable` operation parameters listed here.
+               
+       - **exclusiveStartTableName** : The first table name that the listTables operation evaluates. Use the value returned for LastEvaluatedTableName. 
+       - **limit** : The maximum number of table names to retrieve. If this parameter is not specified, the limit is 100.
+       
+        While invoking the API, the above two parameter values come as a user inputs.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:333]:
```suggestion
        While invoking the API, the above two parameter values come as user inputs.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 17]
```diff
@@ -0,0 +1,945 @@
...
+   
+   You can use the same configuration to initialize the connector. Please follow the steps given in 1.1 for setting up the `init` operation to the addtable operation.
+   
+2. Set up the deleteTable operation.
+
+    1. Navigate into the **Palette** pane and select the graphical operations icons listed under **Amazondynamodb Connector** section. Then drag and drop the `deleteTable` operation into the Design pane.
+           
+       <img src="../../../../assets/img/connectors/amazon-dynamodb-drag-and-drop-delete-table.png" title="Drag and drop list table operation" width="500" alt="Drag and drop list table operation"/>    
+
+    2. The listTables operation use to retrieve information about the created tables. `deleteTable` operation parameters listed here.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:328]:
```suggestion
    2. The listTables operation is used to retrieve information about the created tables. The `deleteTable` operation parameters are listed here.
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 18]
```diff
@@ -0,0 +1,945 @@
...
+    3. Then drag and drop the `Property` mediators into the Design pane as mentioned in `addtable` operation. The parameters available for configuring the Property mediator are as follows.
+        
+       Add the property mediator to capture the `exclusiveStartTableName` value.  Please follow the steps given in `listTables` operation.
+   
+       - **name** : exclusiveStartTableName
+       - **expression** : json-eval($.exclusiveStartTableName)
+        
+        <img src="../../../../assets/img/connectors/amazon-dynamodb-property-mediator-list-table-property1-value1.png" title="Add property mediators to capture key" width="600" alt="Add property mediators to capture exclusiveStartTableName"/>
+       
+    4. Add the property mediator to capture the `limit` values. Please follow the steps given in `listTables` operation.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:309]:
```suggestion
    4. Add the property mediator to capture the `limit` values. Please follow the steps given in the `listTables` operation.               
```

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md [hunk 19]
```diff
@@ -0,0 +1,945 @@
...
+       - **exclusiveStartTableName** : The first table name that the listTables operation evaluates. Use the value returned for LastEvaluatedTableName. 
+       - **limit** : The maximum number of table names to retrieve. If this parameter is not specified, the limit is 100.
+       
+        While invoking the API, the above two parameter values come as a user inputs.
+       
+        <img src="../../../../assets/img/connectors/amazon-dynamodb-drag-and-drop-list-table-parameters.png" title="Drag and drop list table parameter operation" width="500" alt="Drag and drop list table parameter operation"/> 
+    
+    3. Then drag and drop the `Property` mediators into the Design pane as mentioned in `addtable` operation. The parameters available for configuring the Property mediator are as follows.
+        
+       Add the property mediator to capture the `exclusiveStartTableName` value.  Please follow the steps given in `listTables` operation.
```
Comment (samgnaniah) [inline en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:302]:
```suggestion
       Add the property mediator to capture the `exclusiveStartTableName` value.  Please follow the steps given in the `listTables` operation.
```

Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:302]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 18 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:309]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 17 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:328]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 16 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:333]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 15 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:339]: Updated.

File: en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md (hunk 14 above)
Comment (KasunAratthanage) [reply en/micro-integrator/docs/references/connectors/amazondynamodb-connector/amazondynamodb-connector-example.md:346]: Updated.

**Pull request merged**